source/PlayerInfo.h

```diff
@@ -352,6 +352,8 @@ class PlayerInfo {
 
 	// Check that this player's current state can be saved.
 	bool CanBeSaved() const;
+	// Handle the daily salaries and payments.
+	void DoAccounting();
```

Comment:
this should probably be moved to somewhere between line 138 and 144, as that is where the other accounting stuff is

Reply:
That accounting stuff is public, but I think this is better off as private.

Reply:
ah, I overlooked that 👍